specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/applicationGateway.json

```diff
@@ -1895,6 +1920,28 @@
       ],
       "description": "Allows to disable rules within a rule group or an entire rule group."
     },
+    "ApplicationGatewayFirewallExclusion": {
+      "properties": {
+        "matchVariable": {
+          "type": "string",
+          "description": "The variable to be excluded."
+        },
+        "selectorMatchOperator": {
+          "type": "string",
+          "description": "The operation on selector."
+        },
+        "selector": {
+          "type": "string",
+          "description": "The selector to be operated on for the variable to be excluded."
```

Comment:
This is hard to understand and not very helpful. Perhaps more something like this: "When matchVariable is a collection, serlector is used to specify which elements in the collection this exclusion applies to"

Reply:
will change